assets/scripts/fe/src/app/commons/services/auth/user.service.ts

```diff
@@ -30,4 +30,22 @@ export class UserService {
     ;
   }
 
+  hasPass(){
```

Comment:
this is not worth it. just add a boolean field in the user payload and remove this.